habitat/parser.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, config):
 
         config = copy.deepcopy(config)
         parser_config = config["parser"]
-
+        
```

Comment:
Please don't add trailing whitespace

Reply:
I think i've cleaned up all the trailing whitespace now...